src/open_inwoner/utils/ckeditor.py

```diff
@@ -105,3 +118,18 @@ def get_product_rendered_content(product):
                 element.append(icon)
 
     return soup
+
+
+def convert_styling_to_nlds(response: requests.Response) -> None:
+    """
+    Post-process rendered HTML content, replace legacy styling with NLDS classes
+    """
+    raw_html = response.content
+    html_tree = html.fromstring(raw_html)
+
+    for tag, style_class in NLDS_MAP.items():
+        elems = html_tree.cssselect(tag)
+        for item in elems:
+            item.set("class", style_class)
```

Comment:
This clobbers the current class(es). Instead we want to add our class to whatever else might be on there. 

(this is especially a problem when it unintentionally processes the whole page)